Message.h

```diff
@@ -649,7 +649,7 @@ class InfoPeerListMsg : public Message {
 class DeviceInfoMsg : public Message {
 public:
   void init (__attribute__((unused)) const HMID& to,uint8_t count) {
-    Message::init(0x1a,count,0x00,0x00,0x00,0x00);
+    Message::init(0x1a,count,AS_MESSAGE_DEVINFO,0x00,AS_INFO_SERIAL,0x00);
```

Comment:
Not sure if the 0x00 really means AS_INFO_SERIAL here. If I remember right - this is simple zero.